mqt/bench/utils/utils.py

```diff
@@ -308,55 +310,40 @@ def get_molecule(benchmark_instance_name: str):
     return instances[benchmark_instance_name]
 
 
-def postprocess_oqc_qasm_files():
-    directory = get_qasm_output_path()
-    Parallel(n_jobs=-1, verbose=100)(
-        delayed(postprocess_single_oqc_file)(directory, filename)
-        for filename in os.listdir(directory)
-    )
-
+# def postprocess_oqc_qasm_files():
+#     directory = get_qasm_output_path()
+#     Parallel(n_jobs=-1, verbose=100)(
+#         delayed(postprocess_single_oqc_file)(directory, filename)
+#         for filename in os.listdir(directory)
+#     )
 
-def postprocess_single_oqc_file(directory: str, filename: str):
 
-    f = os.path.join(directory, filename)
-    if "oqc_lucy_qiskit" in f or "oqc_qiskit" in f:
-        with open(f) as f:
+def postprocess_single_oqc_file(filename: str):
+    if "mapped_oqc_lucy_qiskit" in filename or "nativegates_oqc_qiskit" in filename:
```

Comment:
This is very specific to the OQC Lucy device and, hence, might require further adaptation in the future if other devices are added. Wouldn't it make more sense to just check whether the filename contains `oqc` and `mapped` or `nativegates`? And then make a distinction between `qiskit` and `tket`. This might also help to eliminate some duplicate code between both of the branches.

Reply:
Done.